pyproject.toml

```diff
@@ -115,7 +115,11 @@ exclude_lines = [
 ]
 
 [tool.coverage.coverage_conditional_plugin.omit]
-"sys_platform == 'win32'" = ["uvicorn/loops/uvloop.py"]
+"sys_platform == 'win32'" = [
+    "uvicorn/loops/uvloop.py",
+    "uvicorn/supervisors/multiprocess.py",
+    "tests/supervisors/test_multiprocess.py",
```

Comment:
Are you doing this just to not add a lot of pragmas?

Reply:
Yes, because of the signaling mechanism on windows, although we can test part of it, coverage can't catch it.